boa_engine/src/object/jsobject.rs

```diff
@@ -33,32 +33,50 @@ pub struct JsObject {
 }
 
 impl JsObject {
-    /// Create a new `JsObject` from an internal `Object`.
-    fn from_object(object: Object) -> Self {
-        Self {
-            inner: Gc::new(GcCell::new(object)),
-        }
+    /// Creates a new ordinary object with its prototype set to the `Object` prototype.
+    ///
+    /// In spec jargon, this is equivalent to calling the
```

Comment:
Small suggestion on the wording 😄 

```suggestion
    /// This is the equivalent to calling the specification's abstract operation [`OrdinaryObjectCreate(%Object.prototype%)`][call].
```

*The suggestion is meant more for the entire sentence, not just the single line.

Reply:
Like it! Changed in the next commit